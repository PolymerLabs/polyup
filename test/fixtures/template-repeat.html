<link rel='import' href='../polymer/polymer.html'>

<polymer-element name='uses-lots-of-template-repeats' noscript>
  <template>
    <template repeat='{{x in xs}}'>
      <template repeat='{{y, idx in x.ys}}'>
        <template if='{{z}}'>
          <span>{{yes hello}}</span>
        </template>
      </template>
    </template>
    <template repeat='{{abcs}}'>
      <span>{{a}}</span> Good
    </template>
  </template>
</polymer-element>
