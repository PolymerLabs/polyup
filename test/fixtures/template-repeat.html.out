<link rel="import" href="../polymer/polymer.html">

<dom-module id="uses-lots-of-template-repeats">
  <template>
    <template is="dom-repeat" items="{{xs}}" as="x">
      <template is="dom-repeat" items="{{x.ys}}" as="y" index-as="idx">
        <template is="dom-if" if="{{z}}">
          <span>{{yes hello}}</span>
        </template>
      </template>
    </template>
    <!--
        TODO(polyup): convert bindings inside this dom-repeat instance below
        from {{foo}} to {{item.foo}}
     -->
    <template is="dom-repeat" items="{{abcs}}">
      <span>{{a}}</span> Good
    </template>
  </template>
</dom-module>
<script>
  Polymer({ is: 'uses-lots-of-template-repeats' });
</script>
